RFS/src/main/java/org/opensearch/migrations/bulkload/worker/DocumentsRunner.java

```diff
@@ -86,8 +91,34 @@ public CompletionStatus onAlreadyCompleted() throws IOException {
 
                 @Override
                 public CompletionStatus onAcquiredWork(IWorkCoordinator.WorkItemAndDuration workItem) {
-                    doDocumentsMigration(workItem.getWorkItem(), context);
-                    return CompletionStatus.WORK_COMPLETED;
+                    var docMigrationMono = setupDocMigration(workItem.getWorkItem(), context);
+                    var latch = new CountDownLatch(1);
+                    var disposable = docMigrationMono.subscribe( lastItem -> {},
```

Comment:
very nit: extra space after the `(`(don't bother with this unless you're in the file anyways)